cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,196 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
```

Comment:
Is it not possible to select these without using classes/test ids?

Reply:
In this case it can be replaced by other selector (thinking of `div[aria-describedby="Receive Tokens Form"]`). However I feel this is a good case where we should be using a `data-testid` to identify the Modal. WDYT?

We should make sense of this to not break the tests on `web-core`

Reply:
I agree with that comment the vast majority of times but please take a closer look at this specific example. What is the unambiguous way to select the Receive Tokens modal? And with this I am taking into account that we can use a different component/library to display the modal.

My opinion on that is that it's ok to use a `data-testid` for this case.

Reply:
We must ensure that the `data-testid` is added in `web-core` then.

Reply:
Exactly! That would be required for these cases